tests/Components/Editors/TrixTest.php

```diff
@@ -11,3 +11,7 @@
 
     expect(blade('<x-trix name="about"/>'))->toMatchSnapshot();
 });
+
+it('can have classes', function () {
+    expect(blade('<x-trix name="about" trixClass="p-2" />'))->toMatchSnapshot();
```

Comment:
Shouldn't this be `trix-class`?

Reply:
Both forms work, but the convention in this case is kebab-case. (https://laravel.com/docs/11.x/blade#casing)

I updated the test.